src/Simulation/Simulators.Tests/Tests.Microsoft.Quantum.Simulation.Simulators.csproj

```diff
@@ -24,7 +33,18 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
-  </ItemGroup>
+  </ItemGroup>  
+  
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <PropertyGroup>
+      <_ExeDir>$(MSBuildThisFileDirectory)TestProjects\QsharpExe\built\</_ExeDir>
+    </PropertyGroup>
+    <ItemGroup>
+      <_ExeFiles Include="$(_ExeDir)*" />
+    </ItemGroup>
+    <MakeDir Directories="$(OutputPath)TestExe" />    
+    <Copy SourceFiles="@(_ExeFiles)" DestinationFolder="$(OutputPath)TestExe" ContinueOnError="true" />
+  </Target>
```

Comment:
So the Q# executable project copies its binaries from the platform-specific build folder to a known folder, and the test project copies those binaries from the known folder to its own build folder? I see what you mean about this solution not being the nicest. I don't know a better way, either, though. :)

Reply:
Yeah... It's such a silly problem - I just need to somehow know where the executable is that I'm calling from the test, and this actually seemed like the best option...